weather_mv/loader_pipeline/ee.py

```diff
@@ -249,6 +253,9 @@ def add_parser_arguments(cls, subparser: argparse.ArgumentParser):
                                help='Skip validation of regions for data migration. Default: off')
         subparser.add_argument('-u', '--use_personal_account', action='store_true', default=False,
                                help='To use personal account for earth engine authentication.')
+        subparser.add_argument('-f', '--force', action='store_true', default=False,
```

Comment:
I like the explanation you have in the README. Make sure it is also found here, so that users know all the context when they invoke `-h,--help`.

Reply:
Updated it 👍